tests/integration/test_rex.py

```diff
@@ -677,3 +677,113 @@ def test_date_functions(c):
             FROM df
             """
         )
+
+
+@pytest.mark.parametrize("gpu", [False, pytest.param(True, marks=pytest.mark.gpu)])
+def test_totimestamp(c, gpu):
+    if gpu:
+        import cudf
+
+    df = pd.DataFrame(
+        {
+            "a": np.array([1203073300, 1406073600, 2806073600]),
+        }
+    )
+    if gpu:
+        df = cudf.from_pandas(df)
```

Comment:
Don't think we need this as the pandas frame should be getting wrapped in a `dask_cudf.DataFrame` through `create_table`:

```suggestion
```